docs/modules/ROOT/examples/storage/SampleHotRestartConfiguration.java

```diff
@@ -30,6 +30,11 @@ public static void main(String[] args) throws Exception{
         cacheConfig.getMerkleTreeConfig().setDepth(12);
         config.addCacheConfig(cacheConfig);
 
+        MapConfig mapConfig = config.getMapConfig("__sql.catalog");
```

Comment:
Sorry I probably miscommunicated here, but users aren't supposed to configure this IMap at all. They're supposed to use the `CatalogPersistenceEnabled` option in the `SqlConfig`
```
config.getSqlConfig().setCatalogPersistenceEnabled(true);
```

In general `__sql.catalog` is an internal object and might be replaced with something else in the future (similar to `__jet.*` internal IMaps), so we definitely don't want users accessing it directly.

Reply:
Updated with correct config.